config/config_s4.sh

```diff
@@ -0,0 +1,24 @@
+#!/bin/bash
+
+# Compiler/MPI combination
+export HPC_COMPILER="intel/18.0.3"
+export HPC_MPI="impi/18.0.3"
+export HPC_PYTHON="miniconda/3.8-s4"
+
+# Build options
+export USE_SUDO=N
+export PKGDIR=pkg
+export LOGDIR=log
+export OVERWRITE=N
+export NTHREADS=8
+export   MAKE_CHECK=N
+export MAKE_VERBOSE=N
+export   MAKE_CLEAN=N
+export DOWNLOAD_ONLY=N
+export STACK_EXIT_ON_FAIL=Y
+export WGET="wget -nv"
+
+# Load these basic modules for S4
+module purge
+module load license_intel/S4
+module load git/2.30.0
```

Comment:
I think you will need to add a line like this:
```
module use /path/to/miniconda/3.8-s4/modulefile
```
to be able to load the miniconda/3.8-s4  modulefile unless it is being installed via hpc-stack, which I don't think we do, yet.

Reply:
On S4, the path to miniconda/3.8-s4 is in `$MODULEPATH` by default, so it's good to go.  Thanks for checking.